tests/core/test_interpreter.py

```diff
@@ -103,3 +103,35 @@ def regex_interpreter():
         response = {"text": "message_text", "token": None, "message_id": "message_id"}
 
         assert query == response
+
+
+async def test_http_interpreter_no_slash_at_end():
```

Comment:
You can use parametrize to reduce code duplication and define only one test (https://docs.pytest.org/en/latest/parametrize.html). We use it quite often in our test. Here is an example: https://github.com/RasaHQ/rasa/blob/9d1afe0c1c06c9c787a30f99e3deb0bc5bf01620/tests/test_utils.py#L75

Reply:
Thanks. Resolved